generateapis.sh

```diff
@@ -328,11 +328,13 @@ generate_api() {
     fi
   fi
 
-  # Record the commit in synth.metadata, using either googleapis or googleapis-discovery
-  # depending on the generator.
-  if [[ "$GENERATOR" == "regapic" ]]
+  if [[ -f $PACKAGE_DIR/synth.ph ]]
```

Comment:
Should `synth.ph` be `synth.py`?

Reply:
Doh - yes indeed. Will fix in the morning.